pkg/compressor/compressor.go

```diff
@@ -80,14 +80,14 @@ func DecompressSnapshot(data io.ReadCloser, compressionPolicy string) (io.ReadCl
 	case ZlibCompressionPolicy:
 		deCompressedData, err = zlib.NewReader(data)
 		if err != nil {
-			logger.Infof("Unable to decompress: %v", err)
+			logger.Errorf("Unable to decompress: %v", err)
```

Comment:
Can you change this message to lowercase?

Reply:
done